package.json

```diff
@@ -85,7 +85,7 @@
     "copy-to-clipboard": "^3.3.2",
     "currency-symbol-map": "^5.1.0",
     "dayjs": "^1.11.6",
-    "embed-api": "https://github.com/wanderwallet/embed-api#7e144282ad0cc7c901b00c5d753bc570a2f61741",
+    "embed-api": "https://github.com/wanderwallet/embed-api#5dbcb8722252cb6a0a7ca6092d93b18d4ac071d0",
```

Comment:
@7i7o I think we need to `yarn install` the dependencies again to reflect it in `yarn.lock` ?